sdv/tabular/base.py

```diff
@@ -135,13 +136,13 @@ def get_metadata(self):
         """
         return self._metadata
 
-    def _sample_rows(self, num_to_sample):
+    def _sample_rows(self, num_to_sample=None, conditions=None):
```

Comment:
I'm thinking that everything will be simpler if we do not make `num_to_sample` be ever `None`, and if `conditions` at this point is always a flat mapping (like a `dict` or a `Series).
That will also simplify a bit how the subclasses handle this input.